src/main/java/net/snowflake/client/core/HttpProtocol.java

```diff
@@ -0,0 +1,19 @@
+package net.snowflake.client.core;
```

Comment:
We are adding copyright header in the first line (look at the other classe) - please add such header with a current year

Reply:
done